### PR TITLE
add linux/unix/windows helper - fix env printer usage of sw_vers

### PR DESCRIPTION
**Pull request by hjanuschka**
fix issues with `sw_ver` in env printer, on non mac OS.

### CODE REVIEW

File: fastlane/lib/fastlane/environment_printer.rb [hunk 1]
```diff
@@ -132,10 +132,24 @@ def self.print_system_environment
...
-      product, version, build = `sw_vers`.strip.split("\n").map { |line| line.split(':').last.strip }
+      product, version, build = "Unkown"
+      os_version = "UNKOWN"
+
+      if FastlaneCore::Helper.mac?
+        product, version, build = `sw_vers`.strip.split("\n").map { |line| line.split(':').last.strip }
+        os_version = `sw_vers -productVersion`.strip
+      end
+
+      if FastlaneCore::Helper.linux?
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/environment_printer.rb]: This has to be a separate PR, as we have to push a _fastlane_core_ release first 👍

**Review by KrauseFx — CHANGES_REQUESTED** [review]

**Comment by hjanuschka** [comment]
alright i am closing this, and doing 2 seperates - the helper stuff - and the env printer
